src/Journalist.EventStore/Streams/IEventStreamConsumer.cs

```diff
@@ -10,10 +10,14 @@ public interface IEventStreamConsumer
 
         Task CommitProcessedStreamVersionAsync(bool skipCurrent);
 
+        Task CommitStreamVersionAsync(StreamVersion streamVersion);
```

Comment:
CommitProcessedStreamVersionAsync causes side effects, like sending notification, and does not allow to set particular version to consumer. Howewer, I agree, that this one breaks abstraction, but I don't see a better way to solve migration issue.